### ISSUE TITLE
Further issues with 0.3.10.2 beta (replacing during drafting / cursor positioning)

### ISSUE THREAD

**Issue by lheydon** [yakisova41/return-youtube-comment-username#64]
Hi **@yakisova41**,

Just found two further issues with **0.3.10.2 beta**. See screenshot illustrating both.

![issue1](https://github.com/yakisova41/return-youtube-comment-username/assets/137272414/983c5bdc-1653-4682-a8ae-feb7d44ea0cb)

1. The username isn't replaced with the handle when drafting a comment, only once it is posted. Although in fairness, I've genuinely forgotten whether this was ever the case or not?

2. A definite brand new issue is that the cursor is now inserted on the left of the username/handle rather than to the right of it. I'm not sure whether this is a cursor positioning glitch within the script, or is perhaps due to the textbox format being incorrectly handled as a right-to-left script?

Thanks in advance.

**Comment by yakisova41** [comment]
Added the ability to display the name while typing in 0.3.11
Probably the issue with the cursor moving to the left is a youtube bug or spec.

**Comment by lheydon** [comment]
1. Confirmed display while typing issue fixed in **0.3.11**.
2. Tested the cursor positioning issue with add-on enabled/disabled and confirmed is a YouTube issue rather than yours 👍
